osu.Game.Rulesets.Tau/Objects/Drawables/DrawableTauHitObject.cs

```diff
@@ -96,14 +96,14 @@ protected override void UpdateInitialTransforms()
 
             Box.FadeIn(HitObject.TimeFadeIn);
             Box.MoveToY(-0.485f, HitObject.TimePreempt);
+            if (HitObject.ShouldFeign)
+                this.Delay(HitObject.TimePreempt / 2).RotateTo(Rotation - 180, 200);
```

Comment:
maybe the delay should be shorter?

Reply:
It could be, but too short makes it a bit less notable. The goal of this is to mislead to a certain degree, a shorter delay kinda minimizes the impact imo.

I'd rather make use of testing and feedback from players to determine the value, since Tau has the playerbase to make that a viable option.